spec/models/preserved_copy_spec.rb

```diff
@@ -2,79 +2,53 @@
 
 RSpec.describe PreservedCopy, type: :model do
   let(:endpoint) { Endpoint.find_by(endpoint_name: 'fixture_sr1') }
-  let!(:preserved_object) do
-    policy_id = PreservationPolicy.default_policy.id
-    PreservedObject.create!(druid: 'ab123cd4567', current_version: 1, preservation_policy_id: policy_id)
+  let(:preserved_object) do
+    PreservedObject.create!(
+      druid: 'ab123cd4567',
+      current_version: 1,
+      preservation_policy_id: PreservationPolicy.default_policy.id
+    )
   end
-  let!(:status) { described_class::VALIDITY_UNKNOWN_STATUS }
-  let!(:preserved_copy) do
-    PreservedCopy.create!(
-      preserved_object_id: preserved_object.id,
-      endpoint_id: endpoint.id,
-      version: 0,
+  let(:status) { 'validity_unknown' }
```

Comment:
I described some of my thinking on the slack channel earlier this week.  
1. The enum strings are part of the class API, because ActiveRecord gives us methods like `preserved_copy.ok!` or `preserved_copy.validity_unknown!` (I use some of these in this PR.)
1. Constants are **not** part the contract of a class, so referencing them from outside the class should be minimized.
2. tests should tend towards the more literal/less indirect when possible

I didn't impose this on every last line, but I did try to do it where it was easy or I was already touching the line, like here, where my main motivation was to convert the `let!` that wasn't always necessary to execute every example.